.github/ISSUE_TEMPLATE/bug-cli.yaml

```diff
@@ -91,7 +92,7 @@ body:
       description: >-
         Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
         Kyverno CLI logs may be found by passing the -v flag to any command.
-      render: shell
+      render: Shell
```

Comment:
Why does this need to be changed?

Reply:
it says `shell` is not a valid value in my LSP